### DISCUSSION TITLE
codemirror-editor-vue3中如何实现代码提示？

### DISCUSSION THREAD

**Discussion by Rawwhite** [rennzhang/codemirror-editor-vue3#21] [Q&A] [answered]
大佬您好！我因为项目需要最新才接触到codemirror，不巧的是现在正处于codemirror5与codemirror6的过渡阶段，我几乎找不到codemirror6相关的案例，而codemirror-editor-vue3的出现真的帮助我省下了很多心思，我惊叹于codemirror-editor-vue3的便利，我引入之后马上就能没有错误的使用。但是我留意到主页中似乎没有很充分地提到拓展功能的使用，比如 如何实现诸如 python/C++这类语言的代码提示，我尝试了vue-codemirror也只能做到高亮的程度，或许是我刚刚接触codemirror还不了解怎么使用，，，但是还是希望大佬有时间的话可以给我一点提示！非常感谢！

**Comment by rennzhang** [comment, marked as answer]
代码提示可能使用[monaco-editor](https://microsoft.github.io/monaco-editor/)更合适

**Reply by Rawwhite** [reply]
再次感谢大佬，了解到codemirror需要自己写代码实现提示功能后，我尝试了一下monaco-editor，只需要调用其中的registerCompletionItemProvider函数便可做到代码提示，比自己写一些代码方便且权威，再次感谢大佬的之路！
